_install-and-configure/configuring-opensearch/cluster-settings.md

```diff
@@ -114,8 +114,16 @@ OpenSearch supports the following cluster-level shard, block, and task settings:
 
     Default is `all`. 
 
-- `cluster.persistent_tasks.allocation.recheck_interval` (Time unit): The cluster manager automatically checks whether persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. Default is `30 seconds`, with a minimum of `10 seconds` being required. 
+- `cluster.persistent_tasks.allocation.recheck_interval` (Time unit): The cluster manager automatically checks whether persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. Default is `30 seconds`, with a minimum of `10 seconds` being required.
 
 ## Cluster-level index settings
 
-For information about index-level index settings, see [Cluster-level index settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index-settings/#cluster-level-index-settings).
\ No newline at end of file
+For information about index-level index settings, see [Cluster-level index settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index-settings/#cluster-level-index-settings).
+
+## Cluster-level coordination settings
+
+OpenSearch supports the following cluster-level coordination settings. All settings in this list are dynamic:
+
+- `cluster.fault_detection.leader_check.timeout` (Time unit): Determines the amount of time a node waits for a response from the cluster manager during a leader check before deeming the check as a failure. 
```

Comment:
We should also mention the default timeout here.

Reply:
+1, let us mention the default and min/max values

Reply:
@niyatiagg you can include `Changing this setting from the default value may cause an unstable cluster.` as the last sentence here.